tests/wpt/web-platform-tests/css/css-backgrounds-3/body-bgcolor-attribute-change.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#attr-body-bgcolor">
+<link rel="match" href="body-bgcolor-attribute-change-ref.html">
+<body bgcolor="yellow">
+  Click me, I should become green.
```

Comment:
I'd avoid the `Click me` text, and replace it with something like:

> Passes if the background is green.

Or something like that.